src/db/controller/client.ts

```
@@ -77,4 +97,38 @@ export class ClientController {
 
     return client ?? this.addClient(rest, chainId, clientId)
   }
+
+  public static getClientsToUpdate(
+    chainId: string,
+    counterpartyChainIds: string[]
+  ): ClientTable[] {
+    const clients = select<ClientTable>(
+      DB,
+      this.tableName,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)